src/css/custom.css

```
@@ -211,3 +216,14 @@ div[title="Expand sidebar"]:hover {
 .navbar__link--active {
   font-weight: bold;
 }
+
+/* Add border to screenshots */
+.theme-doc-markdown img:not(.infographic) {
```

Did we decide to keep the border on all images? If yes then we can remove the `:not` pseudo class:

```suggestion
.theme-doc-markdown img {
```

Or alternatively, we can add a more general class for removing the border. For example:

```suggestion
.theme-doc-markdown img:not(.no-border) {
```

Then add the class for this.

☝🏼 Thinking about this because I saw your comment about the SSO screenshots. IMO those should be cleaned up eventually anyway... but this could be a good temp fix for this!